package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "restaurant-hompage",
+  "version": "1.0.0",
+  "description": "A restaurant homepage",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```

Comment:
configure webpack professionally ![like this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png)
`
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

`

Reply:
Professionalized 😎